governance.md

```diff
@@ -326,15 +326,24 @@ Partners, with associated benefits:
 Changes to the governance documents are submitted via a GitHub pull
 request to The Project's governance documents GitHub repository at
 [https://github.com/jupyter/governance](https://github.com/jupyter/governance).
-The pull request is then refined in response to public comment and
-review, with the goal being consensus in the community.  After this
-open period, a Steering Council Member proposes to the Steering
-Council that the changes be ratified and the pull request merged
-(accepting the proposed changes) or proposes that the pull request be
-closed without merging (rejecting the proposed changes).  The Member
-should state the final commit hash in the pull request being proposed
-for acceptance or rejection and briefly summarize the pull request. All votes
-are limited in time to 4 weeks after the vote is initiated. At the end of 4
+There are two phases to the process:
+
+**The discussion phase** begins when the person first opens a pull-request. During
+this time, *the pull-request must be in a draft state*.
+The pull request is refined in response to public comment and
+review, with the goal being consensus in the community.
+
+The pull request author may *call a vote* when they believe enough feedback and
+iteration has occurred. This is triggered by moving the pull request from the
+*draft state* to an *active state*. This triggers the voting phase.
+
+**The voting phase** begins when the PR enters an active state. The proposed changes
+in the pull request are frozen, and may not be substantively modified after voting
+has begun. During the voting phase, the Steering Council
+votes on whether the changes are ratified and the pull request merged (accepting the proposed changes) or
+that the pull request be closed without merging (rejecting the proposed changes).
```

Comment:
No content changes, just rewrapping the text for better linebreaks in the raw source, for those of us that like to look at markdown document source :).

```suggestion
There are two phases to the process:

**The discussion phase** begins when the person first opens a pull-request.
During this time, *the pull-request must be in a draft state*. The pull
request is refined in response to public comment and review, with the goal
being consensus in the community.

The pull request author may *call a vote* when they believe enough feedback
and iteration has occurred. This is triggered by moving the pull request from
the *draft state* to an *active state*. This triggers the voting phase.

**The voting phase** begins when the PR enters an active state. The proposed
changes in the pull request are frozen, and may not be substantively modified
after voting has begun. During the voting phase, the Steering Council votes on
whether the changes are ratified and the pull request merged (accepting the
proposed changes) or that the pull request be closed without merging
(rejecting the proposed changes).
```